ambry-cloud/src/main/java/com.github.ambry.cloud/azure/AzureCloudDestination.java

```diff
@@ -127,7 +127,7 @@ public boolean uploadBlob(BlobId blobId, long inputLength, CloudBlobMetadata clo
         azureMetrics.backupSuccessByteRate.mark(inputLength);
       }
       return uploaded;
-    } catch (BlobStorageException | DocumentClientException | IOException e) {
+    } catch (Exception e) {
```

Comment:
Use `Exception` to catch RunTimeException?

Reply:
Yes.  Azure client throws IllegalStateException on timeouts.  Given the amount of different exceptions being thrown, I gave up and decided to catch them all.  If we do get a RuntimeException from our own code, we should still see it in the log.